lib/PuppeteerSharp/Page.cs

```diff
@@ -1371,6 +1377,162 @@ public async Task<Response> WaitForNavigationAsync(NavigationOptions options = n
             return responses.GetValueOrDefault(_frameManager.MainFrame.Url);
         }
 
+        /// <summary>
+        /// Waits for a request.
+        /// </summary>
+        /// <example>
+        /// <code>
+        /// <![CDATA[
+        /// var firstRequest = await page.WaitForRequestAsync("http://example.com/resource");
+        /// return firstRequest.Url;
+        /// ]]>
+        /// </code>
+        /// </example>
+        /// <returns>A task which resolves when a matching request was made.</returns>
+        /// <param name="url">URL to wait for.</param>
+        /// <param name="options">Options.</param>
+        public async Task<Request> WaitForRequestAsync(string url, WaitForOptions options = null)
```

Comment:
this method should just call `WaitForRequestAsync(request => request.Url == url)` instead of duplicating the code

Reply:
That's perfect